lab11.md

```diff
@@ -189,6 +189,6 @@ $ echo -n "This is a message" | faas-cli invoke hmac-protected --sign hmac --key
 HMAC validation failed.
 ```
 
-As a follow-up task you could apply HMAC to secure your endpoint on `issue-bot` from [lab 5](https://github.com/openfaas/workshop/blob/7f1b0246376b7e4380bb5874655f28254c15f749/lab5.md)
+As a follow-up task you could apply HMAC to secure your endpoint on `issue-bot` from [lab 5](https://github.com/openfaas/workshop/blob/7f1b0246376b7e4380bb5874655f28254c15f749/lab5.md) (pay attention to the environment variable used by GitHub to pass the secret value).
```

Comment:
@rgee0 do we pass secrets over environmental variables?

Reply:
I read this to be referring to the the incoming GitHub message's `X-Hub-Signature` value which is passed through to a function as an env-var.

I think this needs to be more specific a message in order for it to be helpful for workshop attendees.

Reply:
My idea was not to specifically tell what needs to be done, but rather provide a hint to give some direction to somebody who may get stuck behind this.

But of course it's up to you whether and how you want to have that hint there.

Reply:
The point is more that we should be precise with the terminology.

If it **is** a secret then we would treat it as such, and not give the impression we use env vars for storing secrets.

If it **isn't** a secret then lets not refer to it as a secret.

I think that here we are probably passing a hashed value from the HTTP headers as an environment variable in order to verify that the message hasn't been altered.

Reply:
Hmm, somehow it seemed to me that it was really a secret. Although I don't remember anymore what exactly did I do there - quite some time has passed from that.